### PR TITLE
Miscellaneous mission improvements

### PR DESCRIPTION
**Pull request by Smittytron**
Just some minor fixes and additions from my running list of updates I want to do with missions.

- Fixes tabs in Ant01
- Removes extra spaces in Monster Tank Madness
- Kill pillboxes with script in Soviet01. Barrels killed them in original and we don't want players spending much time strafing pillboxes in a tutorial mission.
- Added a two second delay trigger in Allies05a because two sound notifications were playing on top of each other.
- Corrects wall ownership in Allies05a
- Adds the `"TanyaResuced"` sound notification in the mission Downunder

### CODE REVIEW

File: mods/ra/maps/soviet-01/soviet01.lua [hunk 1]
```diff
@@ -102,6 +102,12 @@ WorldLoaded = function()
 	Trigger.AfterDelay(DateTime.Seconds(2), InsertYaks)
 	Paratroopers()
 	Trigger.OnDamaged(HayHouse, PanicAttack)
+	Trigger.OnKilled(PillboxBarrel1, function()
+		Pillbox1.Kill()
```
Comment (abcdefg30) [inline mods/ra/maps/soviet-01/soviet01.lua]: This needs an `IsDead` check since someone could destroy the box first.

**Review by abcdefg30 — COMMENTED** [review]
Lgtm otherwise.

**Comment by Smittytron** [comment]
Updated

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/ea8a5e41e11ebb763f9f810b68a80445aa4736e9...d8020a6fb402750b28416c36fa6b6b88158c4bc6

**Pull request merged**